slides/writing-methods/index.html

```diff
@@ -43,6 +43,40 @@ <h1>Ruby Monstas</h1>
           </aside>
         </section>
 
+        <section>
+          <section>
+          <h2>Levels of Abstraction</h2>
+            <p class="fragment fade-in">In this course we will learn about 3 levels of abstraction:</p>
```

Comment:
```suggestion
            <p class="fragment">In this course we will learn about 3 levels of abstraction:</p>
```

I recently realized that `fade-in` is the default so I started to omit it.

Reply:
You could remove `fade-in` in the entire document.

Reply:
Didn't know that, thanks!